masterlist.yaml

```diff
@@ -9547,6 +9547,95 @@ plugins:
         util: '[SSEEdit v4.0.2](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
         udr: 24
 
+  - name: 'DWKaPoTunRace.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/40928'
+        name: 'Races of Nirn - Kapo''tun SE'
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/77612'
+        name: 'Races of Nirn - Kapotun Reborn'
+    group: *racesGroup
```

Comment:
This group is for mods that solely or primarily modify vanilla races or classes. The mod only adds new races; it makes no modifications to vanilla records (at least the original doesn't save for changing the name of a holding cell), so it should be left in default:
```suggestion
```